src/hooks/chart.ts

```diff
@@ -349,7 +362,11 @@ export const useChartData = (plot: Plot) => {
       // request for each trace
       plot.traces.map((trace) => {
         // Check for queryPattern(dynamic link) parameter in traces, if not defined then check for uri(static link)
-        if (trace.queryPattern) {
+        if (trace.url_pattern) {
+          const url = createUrlFromPattern(trace.url_pattern);
+          return axios.get(url);
```

Comment:
Can you comment why axios is being used here? Why aren't we using `ConfigService.http.get`?

If you look at `src/utils/config.ts` in chaise, you can see that `http` is defined like this:
```
static get http() {
  return ConfigService._setupDone && ConfigService._server ? ConfigService._server.http : axios;
}
```

and will default to axios if needed